src/app/community/create/create.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-card class="communityCreationForm">
+  <mat-card-title>Community Creation Form</mat-card-title>
+
+  <mat-card-content>
+    <mat-form-field id="communityNameField">
+      <input matInput placeholder="Community Name" [formControl]="nameFormControl" [(ngModel)]="nameText" maxlength="25" >
+      <mat-error *ngIf="nameFormControl.hasError('required')">
+        Name is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field id="communityTitleField">
+      <input matInput placeholder="Community Title" [formControl]="titleFormControl" [(ngModel)]="titleText" maxlength="100">
+      <mat-error *ngIf="titleFormControl.hasError('required')">
+        Community Title is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field id="comDescriptionField">
+      <textarea matInput placeholder="Community Description" [(ngModel)] = "descriptionText"></textarea>
```

Comment:
You can specify the amount of rows in the text area with the property `rows` instead of setting the height elsewhere (ex. `rows="4"`

Reply:
Ok, I've specified text area with rows property. My only concern is that it does not re-size when screen is minimized... So now all input elements inside the form are not responsive but the card can *slightly* change it's height and buttons will move accordingly... I think it doesn't look good. Should we allow the form to be responsive to change in browser size? Or just fix it...

Reply:
Generally, I don't think people are going to put enormous descriptions for their community. As a result, it seems to be a waste to have a huge description box if there are only going to be one or two sentences (remember these are memes after all).  If anything, maybe the side bar message could be a text box since it could have lots of content with the community rules, links, etc...

Current 4k shot has quite a bit of whitespace at the bottom:
![image](https://user-images.githubusercontent.com/11811793/37553046-afc99832-2985-11e8-98c1-527baf8370d8.png)

I think filling up the whitespace with a much larger description box would look very weird, but it is up to you. Making the form responsive doesn't imply that the contents must much the full height of the screen, it should just look "right" on different screens.

Reply:
Maybe it would look better if this was in a dialog instead?